### PR TITLE
added owens comment to readme

### PR DESCRIPTION
**Pull request by owenmeyer3**
I added my comment to the readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# real-estate
\ No newline at end of file
+# real-estate
+
+owen meyer added this comment.
```
Comment (sambarton10) [inline README.md:3]: Good comment.

Comment (heherrin) [reply README.md:3]: Thumbs up

**Review by heherrin — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Heise1990) [reply README.md:3]: hey there

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
-# real-estate
\ No newline at end of file
+# real-estate
```
Comment (Heise1990) [inline README.md:1]: This looks good!

**Review by Heise1990 — APPROVED** [review]

**Pull request merged**